setup.py

```diff
@@ -53,14 +53,18 @@
     packages=find_packages(),
     # adjust the tabbing
     install_requires=[
-        "numpy",
```

Comment:
Why is this required?

Reply:
No idea. Will remove it.

Reply:
I meant the pinning on `numpy` that you introduce in this PR. `numpy` and `scipy` are dependencies, so they should stay here, but the pinning needed to be removed, as there are no reasons to pin them.

Reply:
still open